packages/mobile/src/redux/store.ts

```diff
@@ -1,18 +1,20 @@
 import { applyMiddleware, compose, createStore } from 'redux'
-import { persistReducer, persistStore } from 'redux-persist'
+import { createMigrate, persistReducer, persistStore } from 'redux-persist'
 import autoMergeLevel2 from 'redux-persist/lib/stateReconciler/autoMergeLevel2'
 import storage from 'redux-persist/lib/storage'
 import createSagaMiddleware from 'redux-saga'
 import thunk from 'redux-thunk'
+import { migrations } from 'src/redux/migrations'
 import rootReducer from 'src/redux/reducers'
 import { rootSaga } from 'src/redux/sagas'
 
 const persistConfig: any = {
   key: 'root',
-  version: -1, // default is -1, increment as we make migrations
+  version: 0, // default is -1, increment as we make migrations
   storage,
   blacklist: ['home', 'geth', 'exchange', 'networkInfo', 'alert'],
   stateReconciler: autoMergeLevel2,
+  migrate: createMigrate(migrations, { debug: true }),
```

Comment:
Shouldn't `debug` be false?

Reply:
Debug just adds some logging, lets leave it as true for now:
https://github.com/rt2zz/redux-persist/blob/208c6b112ead87b3701dfacaae2cdbe78377775a/src/createMigrate.js